app/src/main/res/layout/contact_page.xml

```diff
@@ -18,74 +18,72 @@
         android:textAlignment="center"
         android:textColor="#000"
         android:textSize="55sp"
-        android:textStyle="bold"
-        app:layout_constraintBottom_toTopOf="@+id/svContactSearch"
-        app:layout_constraintEnd_toEndOf="parent"
-        app:layout_constraintStart_toStartOf="parent"
-        app:layout_constraintTop_toTopOf="parent" />
+        android:textStyle="bold" />
 
-    <!--Add onclick function-->
-    <androidx.appcompat.widget.SearchView
-        android:id="@+id/svContactSearch"
+    <LinearLayout
+        android:id="@+id/linearLayout4"
         android:layout_width="match_parent"
-        android:layout_height="44dp"
-        android:layout_margin="10dp"
-        android:layout_marginStart="10dp"
-        android:layout_marginTop="1dp"
-        android:layout_marginEnd="9dp"
-        android:layout_marginBottom="10dp"
-        android:background="@drawable/search_bar_background"
-        app:iconifiedByDefault="false"
-        app:layout_constraintBottom_toTopOf="@+id/swipeContainer"
-        app:layout_constraintEnd_toEndOf="parent"
-        app:layout_constraintStart_toStartOf="parent"
-        app:layout_constraintTop_toBottomOf="@+id/textView5"
-        app:queryHint="Search Contact...">
+        android:layout_height="145dp"
+        android:gravity="top"
+        android:orientation="horizontal"
+        app:layout_anchor="@+id/swipeContainer"
+        app:layout_anchorGravity="top|center">
+
+        <androidx.appcompat.widget.SearchView
+            android:id="@+id/svContactSearch"
+            android:layout_width="match_parent"
+            android:layout_height="wrap_content"
+            android:layout_gravity="top"
+            android:layout_margin="10dp"
+            android:background="@drawable/search_bar_background"
+            app:iconifiedByDefault="false"
+            app:queryHint="Search Contact...">
```

Comment:
Can the text for this hints be added to the string.xml file? To make things easier to translate all out text strings

Reply:
just change it in a later pull request.